thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/detector/db/dao/AnomalyFunctionSpecDAO.java

```diff
@@ -7,39 +7,16 @@
 
 import com.linkedin.thirdeye.detector.db.entity.AnomalyFunctionSpec;
 
-import io.dropwizard.hibernate.AbstractDAO;
-
-public class AnomalyFunctionSpecDAO extends AbstractDAO<AnomalyFunctionSpec> {
+public class AnomalyFunctionSpecDAO extends AbstractBaseDAO<AnomalyFunctionSpec> {
   public AnomalyFunctionSpecDAO(SessionFactory sessionFactory) {
     super(sessionFactory);
   }
 
-  public AnomalyFunctionSpec findById(Long id) {
-    return get(id);
-  }
-
-  public Long createOrUpdate(AnomalyFunctionSpec anomalyFunctionSpec) {
-    long id = persist(anomalyFunctionSpec).getId();
-    currentSession().getTransaction().commit();
```

Comment:
We had added these lines "currentSession().getTransaction().commit();" in the createOrUpdate as it was causing failures in AnomalyResource. Do Anomaly Dashboard CRUD operations in particular work with these changes?